### ISSUE TITLE
labgum experiment needs maintenance

### ISSUE THREAD

**Issue by bueler** [pism/pism#475]
**Description**

Just following the manual; see https://pism-docs.org/sphinx/manual/validation/labgum.html.  The `preprocess.py` stage seemed fine but then:
```
$ ./rungum.sh 4 52
Reading configuration parameters (pism_overrides) from file 'gumparams.nc'.
Reading configuration parameters (pism_config) from file '/usr/local/share/pism/pism_config.nc'.
PISM ERROR: unrecognized parameter ice_softness_doc in gumparams.nc
WARNING! There are options you set that were not used!
WARNING! could be spelling mistake, etc!
There are 25 unused database options. They are:
Option left: name:-bootstrap (no value)
Option left: name:-energy value: none
Option left: name:-extra_file value: ex_lab52.nc
Option left: name:-extra_times value: 0:3.1689e-07:2.36555048199e-5
Option left: name:-extra_vars value: diffusivity,flux_mag,velbar_mag,velsurf_mag,mask,thk,wvelsurf
Option left: name:-gradient value: mahaffy
Option left: name:-i value: initlab52.nc
Option left: name:-Lbz value: 0
Option left: name:-Lz value: 0.1
Option left: name:-max_dt value: 3.1689e-09
Option left: name:-Mbz value: 0
Option left: name:-Mx value: 52
Option left: name:-My value: 52
Option left: name:-Mz value: 101
Option left: name:-o value: lab52.nc
Option left: name:-sia_e value: 1.0
Option left: name:-sia_flow_law value: isothermal_glen
Option left: name:-surface value: given
Option left: name:-surface.given.smb_max value: 1e10
Option left: name:-surface_given_file value: initlab52.nc
Option left: name:-ts_file value: ts_lab52.nc
Option left: name:-ts_times value: 3.1689e-09:3.1689e-09:2.36555048199e-5
Option left: name:-y value: 2.36555048199e-5
Option left: name:-ys value: 0.0
Option left: name:-z_spacing value: equal
```

**PISM version**

```
$ pismr -version
Petsc Development GIT revision: v3.14.2-242-g8c55599ccc  GIT Date: 2020-12-04 23:07:41 -0600
       The PETSc Team
    petsc-maint@mcs.anl.gov
 https://www.mcs.anl.gov/petsc/
See docs/changes/index.html for recent updates.
See docs/faq.html for problems.
See docs/manualpages/index.html for help. 
Libraries linked from /home/bueler/petsc/linux-c-dbg/lib
----------------------------------------
Reading configuration parameters (pism_config) from file '/usr/local/share/pism/pism_config.nc'.
PISMR (basic evolution run mode) stable v1.2.2 committed by Constantine Khrulev on 2020-08-28 12:22:53 -0800
PISM (stable v1.2.2 committed by Constantine Khrulev on 2020-08-28 12:22:53 -0800)
CMake 3.16.3.
Petsc Development GIT revision: v3.14.2-242-g8c55599ccc  GIT Date: 2020-12-04 23:07:41 -0600
PETSc configure: --download-mpich --with-debugging=1
MPICH Version:	3.3.2
MPICH Release date:	Tue Nov 12 21:23:16 CST 2019
MPICH ABI:	13:8:1
MPICH Device:	ch3:sock
MPICH configure:	--prefix=/home/bueler/petsc/linux-c-dbg MAKE=/usr/bin/make --libdir=/home/bueler/petsc/linux-c-dbg/lib CC=gcc CFLAGS=-fPIC -fstack-protector -g3 AR=/usr/bin/ar ARFLAGS=cr CXX=g++ CXXFLAGS=-fstack-protector -g -fPIC FFLAGS=-fPIC -ffree-line-length-0 -g FC=gfortran F77=gfortran FCFLAGS=-fPIC -ffree-line-length-0 -g --enable-shared --with-pm=hydra --enable-fast=no --enable-error-messages=all --with-device=ch3:sock --enable-g=meminit
MPICH CC:	gcc -fPIC -fstack-protector -g3   -O0
MPICH CXX:	g++ -fstack-protector -g -fPIC  -O0
MPICH F77:	gfortran -fPIC -ffree-line-length-0 -g  -O0
MPICH FC:	gfortran -fPIC -ffree-line-length-0 -g  -O0
NetCDF 4.7.3 of Jan 23 2020 04:18:16 $.
FFTW fftw-3.3.8-sse2-avx.
GSL 2.5.
```

**To Reproduce**

Please include

1. The *full* command necessary to reproduce the bug.
2. All necessary input files.

It is great if you can
- remove irrelevant command-line options from the command exhibiting the bug
- find a quick run that exhibits the bug by reducing the grid size and the duration of the run
- reduce the size of files it requires,
- provide a definitive way of checking if the bug in question is fixed.

**Expected behavior**

Please provide a clear and concise description of what you expected to happen.

**Additional context**

Additional notes, descriptions of the context, etc should go here.

**Comment by ckhroulev** [comment]
Fixed in the `dev` branch (see 68a3e8dfe6).

**Comment by bueler** [comment]
Works for me.
